### ISSUE TITLE
Validation

### ISSUE THREAD

**Issue by doantientai** [junyanz/CycleGAN#94]
Why don't we have val data and validation loss along with train and test data such like in pix2pix and other networks?

**Comment by junyanz** [comment]
Pix2pix can be evaluated using ground truth input-output pairs. But in the unpaired case, we don't even have the ground truth output for a given input. One can calculate the GAN loss and cycle loss on a validation set. But the loss itself may not tell you much.

**Comment by doantientai** [comment]
Oh I got it. That's right. Thanks!

**Comment by gimletkim** [comment]
@junyanz If the validation loss itseelf may not tell me much, dose it have no meaning? If so, when I split datasets to use cycleGAN, is it unncessary to have validation sets?
